cmake/ECMQueryQmake.cmake

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
This file seems unused? There's also one with the same name in cmake/ECM/modules (with a removed copyright header and using a non-existent include?). Maybe some kind of merge conflict?

Reply:
yes, removed this one as it was recently moved into `cmake/ECM` directory.